meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-tools_2.7.bb

```diff
@@ -25,6 +25,7 @@ do_compile:prepend () {
     sed -e '/^LDLIBS/ s,$, \$\{BUILD_LDFLAGS},' \
         -e '/^INCLUDE_PATHS/ s,$, \$\{BUILD_CFLAGS},' \
         -i ${S}/tools/fiptool/Makefile
+    sed -i '/^LIB/ s,$, \$\{BUILD_LDFLAGS},' ${S}/tools/cert_create/Makefile
```

Comment:
maybe we could use the upstream version on meta-arm some day
https://git.yoctoproject.org/meta-arm/tree/meta-arm/recipes-bsp/trusted-firmware-a/trusted-firmware-a.inc#n166

our fork (forked from st) only have a pending patch that we can carry with a bbappend
https://github.com/foundriesio/meta-lmp/blob/2600b711a3dae66aab28d2fd002e410ab3b98933/meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-tools/0001-tools-allow-to-use-a-root-key-password-from-command-.patch

Reply:
Yup, was thinking the same, we can probably just merge tools into the main tf-a-fio recipe and have it all packaged correctly so it can be reused as native tools for fip and cert_create.

Since this needs more changes, in order to be more aligned with the meta-arm recipes, I would prefer to land this fix and late rework the recipes to reduce the need for this extra one.

Reply:
I think the same and my suggestion is more about something we can do to drop this forks to reduce maintenance effort